Library/Homebrew/cmd/create.rb

```diff
@@ -107,17 +113,20 @@ def url=(url)
         /(.*?)[-_.]?#{path.version}/.match path.basename
         @name = $1
       end
-      @path = Formulary.path @name unless @name.nil?
-    else
-      @path = Formulary.path name
     end
+    update_path
     if @version
       @version = Version.new(@version)
     else
       @version = Pathname.new(url).version
     end
   end
 
+  def update_path
+    return if @name.nil? || @tap.nil?
+    @path = tap.formula_dir/"#{name}.rb"
```

Comment:
This will produce wrong results if `@name` happens to include upper-case letters. For simplicity and consistency with previous behavior, I'd make that just `@path = Formulary.path("#{@tap}/#{@name}")`. (Note also the somewhat inconsistent use of the `tap`/`name` readers and corresponding instance variables in this and the preceding line.)